src/jit/flowgraph.cpp

```diff
@@ -5763,7 +5772,7 @@ void Compiler::fgFindBasicBlocks()
             impInlineInfo->iciCall->gtCallMoreFlags |= GTF_CALL_M_DOES_NOT_RETURN;
```

Comment:
Mean the one in `fgFindBasicBlocks`? Could be less returns in there than `fgFindJumpTargets` sees as unreachable returns could be removed between the two?

The issue I'm trying to resolve is `Compiler::compCompileHelper` (I think its this one) runs much earlier and does pre-screening during crossgen and marks some candidates as non-inlinable early so `fgMakeBasicBlocks` never gets run with inlining and the `GTF_CALL_M_DOES_NOT_RETURN` flag therefore never gets set for these methods.